ctx_test.go

```diff
@@ -5847,6 +5847,20 @@ func Test_GenericParseTypeBoolean(t *testing.T) {
 	}
 }
 
+// go test -run Test_Ctx_Drop -v
+func Test_Ctx_Drop(t *testing.T) {
+	t.Parallel()
+
+	app := New()
+	app.Get("/block-me", func(c Ctx) error {
```

Comment:
Add another handler here that just calls `return`, and test the output

Reply:
please check again